src/mono/msbuild/AppleAppBuilder/AppleAppBuilder.cs

```diff
@@ -34,7 +34,7 @@ public class AppleAppBuilderTask : Task
     /// Path to Mono public headers (*.h)
     /// </summary>
     [Required]
-    public string MonoInclude { get; set; } = ""!;
+    public string MonoRuntimeHeaders { get; set; } = ""!;
```

Comment:
What does ` ""!` achieve here? Is this the "null-forgiving" operator?

Reply:
These properties are guaranteed to be non-null always (due to `[Required]` attribute) the compiler forces me to do inline init to leave them non-nullable. I wonder how @stephentoub solves such cases.